src/NzbDrone.Core/ImportLists/Spotify/SpotifyFollowedArtists.cs

```diff
@@ -32,27 +33,43 @@ public class SpotifyFollowedArtists : SpotifyImportListBase<SpotifyFollowedArtis
         public override IList<ImportListItemInfo> Fetch(SpotifyWebAPI api)
         {
             var result = new List<ImportListItemInfo>();
-            
-            var followed = Execute(api, (x) => x.GetFollowedArtists(FollowType.Artist, 50));
-            var artists = followed.Artists;
+
+            var followedArtists = _spotifyProxy.GetFollowedArtists(this, api);
+            var artists = followedArtists?.Artists;
+
             while (true)
             {
-                foreach (var artist in artists.Items)
+                if (artists == null)
+                {
+                    return result;
+                }
+
+                foreach (var artist in artists?.Items ?? new List<FullArtist>())
```

Comment:
`artists?` you already check for null on this above. 

`.Items ?? new List<FullArtist>()` Digging up some stackoverflow conversations seems to point to some performance degradation (not sure it will matter that much) when using null coalescing in foreach <https://stackoverflow.com/questions/6455311/>, could stick it in above null check done on artists